nni/compression/pytorch/compressor.py

```diff
@@ -604,10 +607,13 @@ class Quantizer(Compressor):
     def __init__(self, model, config_list, optimizer=None, dummy_input=None):
         if isinstance(model, torch.nn.DataParallel):
             model = model.module
+        model_copied = copy.deepcopy(model)
```

Comment:
Why we have to use `deepcopy()` here?

Reply:
Because we use `quantizer-wrapped` model to determine which layer's shapes should be recorded. And in the process of recording shape, some hooks would be registered in the model. So I think it is better to use a copied model to do shape recording.